examples/PlaneCurve.jl

```diff
@@ -1,10 +1,10 @@
 module PlaneCurveModule
 using Oscar, Markdown
-import Base.==
+import Base.:(==)
```

Comment:
Hmm, I made the reverse change in a previous commit because with your version, I got an error. Which Julia version are you using?

Reply:
I use Julia 1.5.1.  I changed for Base.== as you suggest.